varnishgather

```diff
@@ -428,6 +440,46 @@ done
 
 shift $((OPTIND - 1))
 
+if [ -z "$NAME" ]; then
+    VARNISHD_CMD=$(ps -o cmd= -p $(pgrep varnishd))
+    INSTANCE_NAME=$(echo "$VARNISHD_CMD" | grep -oP '(?<=-n )\S+')
+
+    if [ -n "$INSTANCE_NAME" ]; then
+        NAME="-n $INSTANCE_NAME"
+    else
+        INSTANCE_NAME=$(suggest_n_values)
```

Comment:
why not start with that?

Reply:
Covered in the previous two updates I hope.